build_and_release.sh

```diff
@@ -47,12 +47,17 @@ echo "Bash version: $BASH_VERSION"
 
 if "$is_mac"; then
   ( set -x; shasum --version )
-else
+elif "$is_centos"; then
   (
     set -x
     sha256sum --version
     libtool --version
```

Comment:
How are we getting away without libtool on Ubuntu?

Reply:
At least when I ran with this docker image, it failed to find libtool pre-installed. It seems most of the tools using configure bring their own libtool.  Running find _build -name libtool_ brings up quite a few of them.